index.html

```diff
@@ -331,6 +331,7 @@ <h5>Transcode to H.264</h5>
         <dt>-crf <em>18</em></dt><dd>Specifying a lower CRF will make a larger file with better visual quality. For H.264 files being encoded with a 4:2:0 chroma subsampling scheme (i.e., using <code>-pix_fmt yuv420p</code>), the scale ranges between 0-51 for 8-bit content, with 0 being lossless and 51 the worst possible quality.<br>
           If no crf is specified, <code>libx264</code> will use a default value of 23. 18 is often considered a “visually lossless” compression.</dd>
       </dl>
+      <p>By default, this recipe will include two or fewer audio tracks in the output file. If you wish to include more audio tracks, consult the <a href="#stream-mapping">entry on stream mapping</a>.</p>
```

Comment:
Wouldn't the default be one audio track - more specifically, one track of each type - video/audio/subs?

Reply:
Ah, to be honest I just went by what @ablwr wrote in https://github.com/amiaopensource/ffmprovisr/issues/383#issue-464296249
I haven't tested it! But should do ...

Reply:
Just did a quick test, and it seems like just one audio stream is mapped over by default. Would anyone else be able to test and confirm this? Cheers! :)

Reply:
I made a two audio track file with `ffmpeg -f lavfi -i sine -f lavfi -i sine -f lavfi -i testsrc -t 1 -map 0 -map 1 -map 2 2tracks.mov ` and i can confirm that without using -map 0 , only one track made it to output, but here's evidence of both tracks making their way to the output when using map 0  ```ffmpeg -i 2tracks.mov -c:v ffv1 -map 0 whathappens.mov
ffmpeg version N-96643-g2942b00285-g2383021a7a+1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 9.2.0 (Rev2, Built by MSYS2 project)
  configuration:  --disable-autodetect --enable-amf --enable-bzlib --enable-cuda --enable-cuvid --enable-d3d11va --enable-dxva2 --enable-iconv --enable-lzma --enable-nvenc --enable-zlib --enable-sdl2 --enable-ffnvcodec --enable-nvdec --enable-cuda-llvm --enable-libmp3lame --enable-libopus --enable-libvorbis --enable-libx264 --enable-libdav1d --disable-debug --enable-fontconfig --enable-libass --enable-libbluray --enable-libfreetype --enable-libmfx --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libwavpack --enable-libwebp --enable-libxml2 --enable-libzimg --enable-libshine --enable-gpl --enable-avisynth --enable-libxvid --enable-libaom --enable-libopenmpt --enable-version3 --enable-openssl --extra-cflags=-DLIBTWOLAME_STATIC --extra-libs=-lstdc++ --extra-cflags=-DLIBXML_STATIC --extra-libs=-liconv --enable-nonfree
  libavutil      56. 39.100 / 56. 39.100
  libavcodec     58. 67.101 / 58. 67.101
  libavformat    58. 37.100 / 58. 37.100
  libavdevice    58.  9.103 / 58.  9.103
  libavfilter     7. 74.100 /  7. 74.100
  libswscale      5.  6.100 /  5.  6.100
  libswresample   3.  6.100 /  3.  6.100
  libpostproc    55.  6.100 / 55.  6.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '2tracks.mov':
  Metadata:
    major_brand     : qt
    minor_version   : 512
    compatible_brands: qt
    encoder         : Lavf58.37.100
  Duration: 00:00:01.02, start: 0.000000, bitrate: 224 kb/s
    Stream #0:0: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
    Stream #0:1: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s
    Metadata:
      handler_name    : SoundHandler
    Stream #0:2: Video: h264 (High 4:4:4 Predictive) (avc1 / 0x31637661), yuv444p, 320x240 [SAR 1:1 DAR 4:3], 61 kb/s, 25 fps, 25 tbr, 12800 tbn, 50 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      encoder         : Lavc58.67.101 libx264
File 'whathappens.mov' already exists. Overwrite? [y/N] y
Stream mapping:
  Stream #0:0 -> #0:0 (aac (native) -> aac (native))
  Stream #0:1 -> #0:1 (aac (native) -> aac (native))
  Stream #0:2 -> #0:2 (h264 (native) -> ffv1 (native))
Press [q] to stop, [?] for help
[mov @ 00000164e8550900] Using MS style video codec tag, the file may be unplayable!
Output #0, mov, to 'whathappens.mov':
  Metadata:
    major_brand     : qt
    minor_version   : 512
    compatible_brands: qt
    encoder         : Lavf58.37.100
    Stream #0:0: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      encoder         : Lavc58.67.101 aac
    Stream #0:1: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s
    Metadata:
      handler_name    : SoundHandler
      encoder         : Lavc58.67.101 aac
    Stream #0:2: Video: ffv1, yuv444p, 320x240 [SAR 1:1 DAR 4:3], q=2-31, 200 kb/s, 25 fps, 12800 tbn, 25 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      encoder         : Lavc58.67.101 ffv1
frame=   25 fps=0.0 q=-0.0 Lsize=     308kB time=00:00:01.02 bitrate=2473.4kbits/s speed=12.4x
video:288kB audio:18kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.945932%
[aac @ 00000164e85649c0] Qavg: 126.415
[aac @ 00000164e86c1cc0] Qavg: 126.415
```

Reply:
and just to be safe - here's without map 0:

```
ffmpeg -i 2tracks.mov -c:v ffv1  ohno.mov
ffmpeg version N-96643-g2942b00285-g2383021a7a+1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 9.2.0 (Rev2, Built by MSYS2 project)
  configuration:  --disable-autodetect --enable-amf --enable-bzlib --enable-cuda --enable-cuvid --enable-d3d11va --enable-dxva2 --enable-iconv --enable-lzma --enable-nvenc --enable-zlib --enable-sdl2 --enable-ffnvcodec --enable-nvdec --enable-cuda-llvm --enable-libmp3lame --enable-libopus --enable-libvorbis --enable-libx264 --enable-libdav1d --disable-debug --enable-fontconfig --enable-libass --enable-libbluray --enable-libfreetype --enable-libmfx --enable-libmysofa --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-amrwbenc --enable-libwavpack --enable-libwebp --enable-libxml2 --enable-libzimg --enable-libshine --enable-gpl --enable-avisynth --enable-libxvid --enable-libaom --enable-libopenmpt --enable-version3 --enable-openssl --extra-cflags=-DLIBTWOLAME_STATIC --extra-libs=-lstdc++ --extra-cflags=-DLIBXML_STATIC --extra-libs=-liconv --enable-nonfree
  libavutil      56. 39.100 / 56. 39.100
  libavcodec     58. 67.101 / 58. 67.101
  libavformat    58. 37.100 / 58. 37.100
  libavdevice    58.  9.103 / 58.  9.103
  libavfilter     7. 74.100 /  7. 74.100
  libswscale      5.  6.100 /  5.  6.100
  libswresample   3.  6.100 /  3.  6.100
  libpostproc    55.  6.100 / 55.  6.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '2tracks.mov':
  Metadata:
    major_brand     : qt
    minor_version   : 512
    compatible_brands: qt
    encoder         : Lavf58.37.100
  Duration: 00:00:01.02, start: 0.000000, bitrate: 224 kb/s
    Stream #0:0: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
    Stream #0:1: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s
    Metadata:
      handler_name    : SoundHandler
    Stream #0:2: Video: h264 (High 4:4:4 Predictive) (avc1 / 0x31637661), yuv444p, 320x240 [SAR 1:1 DAR 4:3], 61 kb/s, 25 fps, 25 tbr, 12800 tbn, 50 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      encoder         : Lavc58.67.101 libx264
Stream mapping:
  Stream #0:2 -> #0:0 (h264 (native) -> ffv1 (native))
  Stream #0:0 -> #0:1 (aac (native) -> aac (native))
Press [q] to stop, [?] for help
[mov @ 0000018dd8680740] Using MS style video codec tag, the file may be unplayable!
Output #0, mov, to 'ohno.mov':
  Metadata:
    major_brand     : qt
    minor_version   : 512
    compatible_brands: qt
    encoder         : Lavf58.37.100
    Stream #0:0: Video: ffv1, yuv444p, 320x240 [SAR 1:1 DAR 4:3], q=2-31, 200 kb/s, 25 fps, 12800 tbn, 25 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      encoder         : Lavc58.67.101 ffv1
    Stream #0:1: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 69 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      encoder         : Lavc58.67.101 aac
frame=   25 fps=0.0 q=-0.0 Lsize=     299kB time=00:00:01.02 bitrate=2395.1kbits/s speed=14.3x
video:288kB audio:9kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.680355%
[aac @ 0000018dd6e55d40] Qavg: 126.415
```

Reply:
Yep, that matches what I found: no mapping -> just one audio track in the output.
I'll update the change to talk about >1 audio track rather than >2.

Reply:
Updated! Could I please get a re-review, @kieranjol?